postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresSchemaReader.java

```diff
@@ -38,6 +38,17 @@ public class PostgresSchemaReader extends CommonSchemaReader {
     "bit", "timetz", "money"
   );
 
+  private String sessionID;
+
+  public PostgresSchemaReader() {
+    super();
```

Comment:
similar comment, call the other constructor

Reply:
Fixed, thanks!